### PR TITLE
Beacon edit

### PR DESCRIPTION
**Pull request by zhekson1**
Removed much of the motivation section for inclusion in cardano-swaps and CPS. Motivation now only for Beacon CIP + Aesthetic changes

### CODE REVIEW

**Comment by fallen-icarus** [comment]
You don't have any mention of delegation control in the abstract or motivation. You instead focus on concurrency. But then in the rational section, the reverse is true. Full delegation control should be mentioned as a major motivation, too.

File: CIP-????/README.md [hunk 1]
```diff
@@ -13,101 +13,101 @@ License: CC-BY-4.0
...
+This CIP outlines a native token standard, called Beacon Tokens, that serves as a way to "tag" on-chain data in a way that is efficiently queryable by off-chain APIs. This enables the creation of **massively** concurrent dApps that require contextual availability of on-chain data, such as programmable P2P swaps.
```
Comment (fallen-icarus) [inline CIP-????/README.md]: I'm not sure **massively** is appropriate, especially given the redundant execution problem. I think **organically** would be better.

**Comment by zhekson1** [comment]
Beacon tokens are a far more generalizable feature than delegation control. Delegation control is a specific feature of dApps that should be relegated to the motivation sections of those dApps.

File: CIP-????/README.md (hunk 1 above)
Comment (zhekson1) [reply CIP-????/README.md]: !!!!

**Comment by fallen-icarus** [comment]
> Delegation control is a specific feature of dApps that should be relegated to the motivation sections of those dApps.

I disagree since we are dealing with PoS. It doesn't need to be the main focus but the fact that beacon tokens mean *all* dapps can have full delegation control is very important.

**Comment by fallen-icarus** [comment]
The rational section is also meant for you to explain how your CIP solved the issues in the motivation. Yet they are currently talking about two different things.

**Comment by zhekson1** [comment]
will revise

**Pull request closed without merging**